ecms_base/ecms_base.profile

```diff
@@ -58,3 +58,34 @@ function ecms_base_update_9013(array &$sandbox): void {
   \Drupal::service('module_installer')->install(['svg_image']);
 
 }
+
+/**
+ * Updates to run for the 0.1.4 tag.
+ */
+function ecms_base_update_9014(array &$sandbox): void {
+  $path = \Drupal::service('extension.list.profile')->getPath('ecms_base');
+
+  /** @var \Drupal\Core\Config\FileStorage $install_source */
+  $install_source = new FileStorage($path . "/config/install/");
+
+  /** @var \Drupal\Core\Config\StorageInterface $active_storage */
+  $active_storage = \Drupal::service('config.storage');
+
+  // Make sure the pathauto and redirect modules are installed.
+  \Drupal::service('module_installer')->install(['pathauto']);
```

Comment:
Instead of calling the `\Drupal::service()` 6 times, it might be slightly more efficient to set it to a variable and use the variable 6 times. Or, the install method takes an array of module names. Maybe just call it once?

Reply:
Good call!